.travis.yml

```diff
@@ -1,14 +1,24 @@
-sudo: required
 dist: trusty
+install: true
 language: java
 jdk:
-- oraclejdk8
-addons:
-  apt:
-    package: maven tar
+    - oraclejdk8
+
+env: DISPLAY=:99.0
+
+before_install:
+    - sudo apt-get install at-spi2-core
+
 before_script:
-- cd ../CodeCheckerEclipsePlugin/eclipse-plugin/
-script: mvn package
+    - sh -e /etc/init.d/xvfb start
+    - sleep 10 # give xvfb some time to start
+
+script: mvn -f mavendeps/pom.xml p2:site && mvn clean verify
+
+after_failure:
+    - cat /home/travis/build/Ericsson/CodeCheckerEclipsePlugin/tests/org.codechecker.eclipse.plugin.rcp.it.tests/target/work/data/.metadata/.log
```

Comment:
Is there and environment variable for the travis build dir? It would be better instead of hard coding the path.